app/dashboard/src/modals/SetupOrganizationAfterSubscribe.tsx

```diff
@@ -69,10 +69,56 @@ export function SetupOrganizationAfterSubscribe() {
   const shouldSetOrgName = PLANS_TO_SPECIFY_ORG_NAME.includes(userPlan) && organizationName === ''
   const shouldSetDefaultUserGroup =
     PLANS_TO_SPECIFY_ORG_NAME.includes(userPlan) && userGroupsCount === 0
-  const shouldShowModal = userIsAdmin && (shouldSetOrgName || shouldSetDefaultUserGroup)
+
+  const steps = [
```

Comment:
nice. i guess this way it's less clear (visually) where all the steps are added... but i do prefer how the data/UI is grouped this way

of course one alternative is to `...shouldSetOrgName ? [{}] : []` so that everything is all in one place but of course, it's a tradeoff in readability

Reply:
> of course one alternative is to ...shouldSetOrgName ? [{}] : [] so that everything is all in one place but of course, it's a tradeoff in readability

Yeah, it was my first implementation, but readability is worse compared to the non-declarative approach.